apps/cms.c

```diff
@@ -186,6 +187,8 @@ const OPTIONS cms_options[] = {
      "Don't include signer's certificate when signing"},
     {"noattr", OPT_NOATTR, '-', "Don't include any signed attributes"},
     {"nosmimecap", OPT_NOSMIMECAP, '-', "Omit the SMIMECapabilities attribute"},
+    {"no_signing_time", OPT_NO_SIGNING_TIME, '-',
```

Comment:
Seems slightly inconsistent naming with other `no` flags (the undersores :) ) - `noattr`,  `nosmimecap`... So maybe `nosigningtime` would be more consistent. But they are some other with underscores so one could choose both. I don't really mind - just mentioning that if it was considered...

Reply:
Well, naming is one of the hard problems of computer science :)
I did consider both versions. I eventually decided on the underscores version because it is consistent with the API flag, and as you say, there are already other options with underscores in them.